File: Basics_with_JavaScript/56 Conditional Statements/More Exersises/02. Sleepy Tom Cat.js
function vladi(input) {
  let nonWorkDays = Number(input);
  let workingDays = 365 - nonWorkDays;
  let tomNeeds = 30000;
  let totalNonWorkDayPlay = nonWorkDays * 127;
  let totalWorkDayPlay = workingDays * 63;
  let totalPlay = totalNonWorkDayPlay + totalWorkDayPlay;
  let difference = Math.abs(totalPlay - tomNeeds);
  let H = Math.floor(difference / 60);
  let M = difference % 60;
  if (totalPlay > tomNeeds) {
    console.log(`Tom will run away`);
    console.log(`${H} hours and ${M} minutes more for play`);
  } else {
    console.log(`Tom sleeps well`);
    console.log(`${H} hours and ${M} minutes less for play`);
  }
}

vladi([20]);
vladi([113]);

File: Basics_with_Python/PythonBasicCourse/2_Conditional_Statements/6_Conditional_Statements_Excercises_Exercise/more_exercises/02._Sleepy_Tom_Cat.py
rest_days = int(input())
max_playtime = 30000
work_days = 365 - rest_days
workday_playtime = 63 * work_days
restday_playtime = 127 * rest_days
total_playtime = workday_playtime + restday_playtime
if total_playtime > max_playtime:
    print(f"Tom will run away")
    print(f"{(total_playtime - max_playtime) // 60} hours and {(total_playtime - max_playtime) % 60} minutes more for play")
else:
    print(f"Tom sleeps well")
    print(f"{(max_playtime - total_playtime) // 60} hours and {(max_playtime - total_playtime) % 60} minutes less for play")
    